e2e/test/E2ETests.csproj

```diff
@@ -10,8 +10,14 @@
     <RootDir>$(MSBuildProjectDirectory)\..\..</RootDir>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>NU1605,NU1902,NU1903</NoWarn>
```

Comment:
these are warnings as errors and are regarding older version of newtonsoft being used.
latest version is 13 and we use 12